code/modules/projectiles/projectile/bullets.dm

```diff
@@ -128,10 +128,10 @@
 	damage = 25
 
 /obj/item/projectile/bullet/pistol/strong
-	damage = 60
-
+	damage = 28
+	armor_penetration = 15
 /obj/item/projectile/bullet/pistol/revolver
-	damage = 45
+	damage = 28
 	armor_penetration = 15
```

Comment:
These are two different calibers, why do they do the same damage? Moreover, this seems like an extreme nerf. 30-35 is fine but 28 is far too little.

Reply:
I will buff the pistol/strong bullet up to 30 or 32, as it is supposed to represent .50 AE. To be frank, pistol/revolver desperately needed a nerf. If necessary I'll bring it up to 30.

Reply:
both need buffs imo